lib/discordrb/data/user.rb

```diff
@@ -166,7 +166,18 @@ def webhook?
       @discriminator == Message::ZERO_DISCRIM
     end
 
-    %i[offline idle online].each do |e|
+    # @!group Statuses
+    # @!method offline?
+    #   @return [true, false] whether this user is offline.
+    # @!method idle?
+    #   @return [true, false] whether this user is idle.
+    # @!method online?
+    #   @return [true, false] whether this user is online.
+    # @!method dnd?
+    #   @return [true, false] whether this user is on Do Not Disturb.
+    # @!endgroup
```

Comment:
```suggestion

    # @!endgroup
```

If we do keep the group directive, the beginning and end must be in separate declarations for YARD to render it properly